Pd 408 review accounts users and groups content (#2029)
truenas/documentation: content/SCALE/SCALETutorials/Credentials/ManageLocalUsersSCALE.md

Enter and confirm the admin user passwords.

Select the **builtin_administrators** group from the **Auxiliary Group** dropdown list.

{{< trueimage src="/images/SCALE/22.12/AddingAdminUserAuxiliaryGroup.png" alt="Add Admin User to builtin_administrators" id="2: Add Admin User to builtin_administrators" >}}

Scroll down to the **Authentication** settings and select the proper **Allow sudo** authorization settings.
Some applications, such as [Nextcloud]({{< relref "InstallNextCloudMedia.md" >}}), require sudo permissions for the administrator account.

> By default, TrueNAS SCALE sets admin account authorization to **Allow all sudo commands**.

Reviewer: No it does not automatically set this for the admin unless the admin is created during the iso installation process. We need to be careful about saying this as a default because someone upgrading to SCALE Bluefin or migrating from CORE does not necessarily have the admin account created for them and they must choose the sudo settings.
Also, it is select the option not click the checkbox.

Author: Ok, but I think we should somehow still communicate that this is the automated permissions default. What about:

```suggestion
For administrator accounts generated during the initial installation process, TrueNAS SCALE sets authorization to **Allow all sudo commands**.
```

Click the checkbox to choose this configuration.

Click **Save**.

> Log out of the TrueNAS system and then log back in using the admin user credentials. Once you are back in the TrueNAS web UI, determine that the admin user credentials are working properly with your network configuration.

Reviewer: After not once, or when not once.
```suggestion
Log out of the TrueNAS system and then log back in using the admin user credentials to verify that the admin user credentials work properly with your network configuration.
```

## Creating User Accounts

To create a new user, click **Add**.

### Configuring User Identification Settings

{{< trueimage src="/images/SCALE/23.10/AddUserIdentificationSettings.png" alt="Add User Identification Settings" id="3: Add User Identification Settings" >}}

> Enter the user's full name in **Full Name**.

Reviewer: do not use the possessive case
```suggestion
Enter the user full name in **Full Name**.
```

Author: Ok. I find the original unclear, so I'm working on options with Tim

By default, the **Disable Password** toggle is not enabled. In this case, set and confirm a password.

Setting **Disable Password** toggle to active (blue toggle) disables several options:

* The **Password** field becomes unavailable and TrueNAS removes any existing password from the account.
* The **Lock User** option disappears.
* The account is restricted from password-based logins for services like SMB shares and SSH sessions.

### Configuring User ID and Groups Settings

{{< trueimage src="/images/SCALE/23.10/AddUser-UserIDAndGroupSettings.png" alt="Add User ID and Groups Settings" id="4: Add User ID and Groups Settings" >}}

Next, you must set a user ID (UID).
TrueNAS suggests a user ID starting at **3000**, but you can change it if you wish.
We recommend using an ID of 3000 or greater for non-built-in users.

By default, TrueNAS creates a new primary group with the same name as the user. This happens when the **Create New Primary Group** toggle is enabled.
To add the user to an existing primary group instead, disable the **Create New Primary Group** toggle and search for a group in the **Primary Group** field.

> You can add the user to more groups using the **Auxiliary Groups** drop-down list.

Reviewer: dropdown not drop-down
```suggestion
You can add the user to more groups using the **Auxiliary Groups** dropdown list.
```

### Configuring Directories and Permissions Settings

{{< trueimage src="/images/SCALE/23.10/AddUserHomeDirPermSCALE.png" alt="Add User Home Directory" id="5: Add User Home Directory" >}}

When creating a user, the home directory path is set to <file>/nonexistent</file>, which does not create a home directory for the user.
To set a user home directory, enter a path in **Home Directory** or select it using the file browser.
If the directory exists and matches the username, TrueNAS sets it as the user home directory.

> When the path does not end with a sub-directory matching the username, TrueNAS creates a new sub-directory if the **Create Home Directory** checkbox is enabled.

Reviewer: no need to mention checkbox when you can communicate this without it
```suggestion
When the path does not end with a sub-directory matching the username, TrueNAS creates a new sub-directory if **Create Home Directory** is selected.
```

TrueNAS shows the path to the user home directory when editing a user.

Select the **Home Directory Permissions** (**Read**, **Write**, **Execute**) for each (**User**, **Group**, **Other**) to set Unix permissions for the user home directory. Built-in users are read-only and can not modify these permissions settings.

### Configuring Authentication Settings

{{< trueimage src="/images/SCALE/23.10/AddUserHomeDirAuthSCALE.png" alt="Add User Home Directory and Authentication Settings" id="6: Add User Home Directory and Authentication Settings" >}}

You can assign a public SSH key to a user for key-based authentication by entering or pasting the *public* key into the **Authorized Keys** field.

{{< hint type=important >}}
Do *not* paste the private key.
{{< /hint >}}

You can also click **Choose File** under **Upload SSH Key** and browse to the location of an SSH key file.

If you are using an SSH public key, always keep a backup of the key.

> Select the [shell]({{< relref "LocalUsersScreensSCALE.md" >}}) option for the user from the **Shell** dropdown options.

Reviewer: option mentioned too many time so maybe this
```suggestion
Select the [shell]({{< relref "LocalUsersScreensSCALE.md" >}}) option for the user from the **Shell** dropdown list.
```

Options are **nologin**, **bash**, **rbash**, **dash**, **sh**, **tmux**, and **zsh**.

> Admin users can also use **TrueNAS CLI** to open shell in the TrueNAS CLI and **TrueNAS Console** to open in the Console Setup Menu for SCALE.

Reviewer: so maybe just describe these new options on the list this way:
```suggestion
Select **TrueNAS CLI** to open shell in the TrueNAS CLI and **TrueNAS Console** to open in the Console Setup Menu for SCALE.
```

Author: I want to specify that you'll only see these options if for admin or root users though. I'll think about it.

Selecting **Lock User** disables all password-based functionality for the account until you clear the checkbox.

> **Allowed sudo commands**, **Allow all sudo commands**, **Allowed sudo commands with no password** and **Allow all sudo commands with no password** allow the account to act as root using the [sudo](https://www.sudo.ws/) command.

Reviewer: Same as previous comment about FIPS, we can say with root-like permissions but we don't want to mention root this way.

> To allow sudo commands, use **Allowed sudo commands** or **Allowed sudo commands with no password** to list the executable path(s) to specific sudo commands allowed for this user.

Reviewer: Okay, this one bothers me a bit, are you saying the command lists the executables or are you saying they can enter the executable paths to command?  This will be very hard for a non-native speaker to understand. Use simple English here. If what they gain is the ability to enter commands or paths say that in as simple a way as possible.

> <file>/usr/bin/</file> is the default location for command executables.

Reviewer: just say commands or commands they can execute which is not really necessary.
```suggestion
<file>/usr/bin/</file> is the default location for commands.
```

> Alternately, click the checkbox for **Allow all sudo commands** or **Allow all sudo commands with no password**.

Reviewer: Simplify this a bit
```suggestion
Or click **Allow all sudo commands** or **Allow all sudo commands with no password**.
```

Exercise caution when allowing sudo commands, especially without password prompts.

> It is recommended to limit this privilege to trusted users and specific commands to minimize security risks.

Reviewer: ```suggestion
We recommend limiting this privilege to trusted users and specific commands to minimize security risks.
```
